projects/jamaica/.env

```diff
@@ -0,0 +1,3 @@
+COMPOSE_PROJECT_NAME=peliasjamaica
```

Comment:
All our projects use `COMPOSE_PROJECT_NAME=pelias`, can we please change this to be consistent with the other projects?